plugins/table/dialogs/table.js

```diff
@@ -193,7 +193,11 @@
 									newCell.removeAttribute( 'scope' );
 								}
 							}
-							theRow.insertBefore( previousFirstRow );
+							if ( previousFirstRow ) {
+								theRow.insertBefore( previousFirstRow );
+							} else {
+								tbody.append( theRow );
+							}
```

Comment:
I think this whole block could be simplified. If you take a look at the `append()` method:

https://github.com/ckeditor/ckeditor-dev/blob/b6733c53a671a17a10324f3c3d333963293477cc/core/dom/element.js#L271-L281

it does very similar think what this code now. So probably we could go with something like:

```js
// If tbody is empty append theRow, if not, insert it before tbody first child.
tbody.append( theRow, tbody.getChildCount() )
```

instead of entire `if/else` and `var previousFirstRow = tbody.getFirst();` line.

Reply:
Actually it can be even simpler, we can always call `append` with `toStart=true`.

According to: https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore#Syntax
> If referenceNode is null, the newNode is inserted at the end of the list of child nodes.

Reply:
Nice, even simpler and shorter 👍